.github/workflows/ci-nym-vpn-cli.yml

```diff
@@ -13,21 +13,22 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  AGENT_ISSELFHOSTED: 1 # https://github.com/actions/setup-go/issues/432
 
 jobs:
   build:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-22.04-arc, custom-runner-mac-m1, windows-2022 ]
-        target: [ native ]
+        os: [ubuntu-22.04-arc, custom-runner-mac-m1, windows11]
```

Comment:
Is it possible to tweak the label to `custom-windows-11`?

Reply:
Yes